src/iden3comm/utils/contract-request.utils.ts

```diff
@@ -0,0 +1,66 @@
+import { keccak256 } from 'ethers';
+import { byteEncoder, hexToBytes, isEthereumIdentity } from '../../utils';
+import { CircuitId } from '../../circuits';
+import { Hex } from '@iden3/js-crypto';
+import { DID } from '@iden3/js-iden3-core';
+import { IProofService } from '../../proof';
+import { ZeroKnowledgeProofResponse } from '../types';
+
+/**
+ * Retrieves the AuthV2 request ID.
+ *
+ * @returns The AuthV2 request ID.
+ */
+export function getAuthV2RequestId(): number {
```

Comment:
maybe we can calculate it and put as constant

Reply:
also we can keep a function and pass circuit id here.

Reply:
fixed